cmd/ethrex/initializers.rs

```diff
@@ -434,6 +435,8 @@ pub async fn init_l1(
         init_metrics(&opts, tracker.clone());
     }
 
+    raise_fd_limit()?;
```

Comment:
Should we log and ignore any errors instead? I'm thinking this could fail with weird setups.